pandas/tests/indexing/multiindex/test_partial.py

```diff
@@ -169,28 +169,6 @@ def test_getitem_intkey_leading_level(
             with tm.assert_produces_warning(FutureWarning):
                 mi.get_value(ser, 14)
 
-    # ---------------------------------------------------------------------
-    # AMBIGUOUS CASES!
-
-    def test_partial_loc_missing(self, multiindex_year_month_day_dataframe_random_data):
```

Comment:
This was original an 'ix' test 10 years ago, never got enabled.  https://github.com/pandas-dev/pandas/blame/1f00335b51d345d58e9574c2c5d86613214aba9b/pandas/tests/test_multilevel.py#L1421

Reply:
not me, hahaha

Reply:
im proud of tracking this one down!

Reply:
heheh no doubt!